### DISCUSSION TITLE
Citizens pathfinding not working

### DISCUSSION THREAD

**Discussion by LHenneveld** [Keriew/augustus#1049] [Q&A]
Hi community! I'm very happy with Augusts and enjoy a challenge, therefore, I'm playing on very hard and without global labour. Because of that, I'm always building housing blocks near my production building which creates a very natural city.

However, the citizens that try to find workers living nearby seem to have a hard time finding their way to the housing nearby. For example, below you see that the blue highlighted walking range of the warehouse isn't going near my housing. I'm expecting it to as it's not that far and other building reach the housing easily. Is this a misunderstanding on my part or is this not working as intended?

Thank you! <3

![image](https://github.com/Keriew/augustus/assets/96427753/d4110569-b2e8-4a63-a83c-b4b3401e4f34)

**Comment by crudelios** [comment]
Hi!

Yes, this is the intended mechanic.

When spawning, the citizen roams freely and does not spacifically look for a housing destination. The blue path denotes which tiles will be visited during free roaming, to help you assist query the distance to the houses.
